django_netjsongraph/tests/test_link.py

```diff
@@ -11,6 +11,14 @@ class TestLink(TestLinkMixin, CreateGraphObjectsMixin, TestCase):
     link_model = Link
 
     def setUp(self):
+        address1 = self.address_model('192.168.0.1')
+        address1.save()
+        address2 = self.address_model('192.168.0.2')
+        address2.save()
         t = self._create_topology()
-        self._create_node(label="node1", addresses="192.168.0.1;", topology=t)
-        self._create_node(label="node2", addresses="192.168.0.2;", topology=t)
```

Comment:
why don't you try put the logic to create the addresses in `_create_node`? That means `_create_node' should still accept an `addresses` argument, it could be a list of addresses

Reply:
Done